### ISSUE TITLE
GoFeed parses the CoinDesk feed but the Link fields are empty

### ISSUE THREAD

**Issue by wjkoh** [mmcdole/gofeed#192]
### Steps to reproduce the behavior

Note: Please include any links to problem feeds, or the feed content itself!

GoFeed parses https://www.coindesk.com/arc/outboundfeeds/rss/ successfully but it returns empty Link fields for both Feed and Item. It's probably due to the multiple Link tags in the feed data. See a couple of examples below.

Channel:
``` 
<channel>
        <title>CoinDesk</title>
        <link>https://www.coindesk.com</link>
        <link href="https://www.coindesk.com/arc/outboundfeeds/rss/?outputType=xml" rel="self" type="application/rss+xml"/>
        <link href="https://pubsubhubbub.appspot.com/" rel="hub"/>
        <atom:link href="https://www.coindesk.com/arc/outboundfeeds/rss/?outputType=xml" rel="self" type="application/rss+xml"/>
        <description>Latest headlines from Coindesk.</description>
```

Item:
```
<item>
            <title>
                <![CDATA[Panic Grips SOL With Record Volatility and Massive Put Demand ]]>
            </title>
            <link>https://www.coindesk.com/markets/2022/11/10/sol-market-in-the-state-of-panic-record-volatility-and-put-demand-suggests/?utm_medium=referral&amp;utm_source=rss&amp;utm_campaign=headlines</link>
            <link href="https://www.coindesk.com/arc/outboundfeeds/rss/?outputType=xml" rel="self" type="application/atom+xml"/>
            <link href="https://pubsubhubbub.appspot.com/" rel="hub"/>
            <guid isPermaLink="false">HAENGWQ2FVEE5OCLQIWC2CQBV4</guid>
```

**Comment by mmcdole** [comment]
Thanks for this report.  Let me create a small test with multiple link tags and inspect the result.

**Comment by mohamed-adly** [comment]
any updates regarding this issue?

**Comment by mmcdole** [comment]
@mohamed-adly @wjkoh this should be fixed in v1.2.0.

Let me know!

**Comment by mohamed-adly** [comment]
That did the trick thank you
